cardano-node/src/Cardano/CLI/Benchmarking/Tx/NodeToNode.hs

```diff
@@ -204,15 +206,16 @@ benchmarkConnectTxSubmit
   -> TxSubmissionClient (GenTxId blk) (GenTx blk) m ()
   -- ^ the particular txSubmission peer
   -> m ()
-benchmarkConnectTxSubmit trs nc localAddr remoteAddr myTxSubClient = do
+benchmarkConnectTxSubmit trs nc localAddr remoteAddr myTxSubClient = withIOManager $ \iocp -> do
```

Comment:
I think this is wrong place to call `withIOManager`.  It should be started at the very top level when the application starts.  We don't want two IO manager threads to complete.